scanner/src/main/kotlin/scanners/BoyterLc.kt

```diff
@@ -87,28 +98,36 @@ object BoyterLc : LocalScanner() {
         }
     }
 
+    override fun getConfiguration() = configuration.joinToString(" ")
```

Comment:
How about making this the default implementation in `LocalScanner` and requiring subclasses to implement a configuration property as a list instead? Would that be to inflexible?

Reply:
Currently only `ScanCode` requires a different implementation, but having a field called `configuration` could be confusing. I'd keep it as is and check again when we add more scanners.

Reply:
👍